auth.py

```diff
@@ -0,0 +1,80 @@
+import functools
+
+from flask import (
+    Blueprint, flash, g, redirect, render_template, request, session, url_for, jsonify
+)
+from werkzeug.security import check_password_hash, generate_password_hash
+
+from db import get_db
+
+bp = Blueprint('auth', __name__, url_prefix='/auth')
+
+@bp.route('/register', methods=["POST"])
+def register():
+    username = request.form['username']
+    password = request.form['password']
+    db = get_db()
+
+    if not username:
+        return jsonify({'status': 'Username is required.'}), 403
+    elif not password:
+        return jsonify({'status': 'Password is required.'}), 403
+
+    try:
+        db.execute(
+            "INSERT INTO User (username, password) VALUES (?, ?)",
+            (username, generate_password_hash(password)),
+        )
```

Comment:
why use raw sql queries in the controller? didn't we agree on having services and repositories?